tgui/packages/tgui/interfaces/NtosSportsBetting.tsx

```diff
@@ -0,0 +1,262 @@
+import { useState } from 'react';
+import {
+  Box,
+  Button,
+  Collapsible,
+  Divider,
+  Icon,
+  Input,
+  NumberInput,
+  Section,
+  Stack,
+  TextArea,
+} from 'tgui-core/components';
+import { BooleanLike } from 'tgui-core/react';
+
+import { useBackend } from '../backend';
+import { NtosWindow } from '../layouts';
+
+type Data = {
+  active_bets: ActiveBets[];
+  bank_name: string;
+  bank_money: number;
+  can_create_bet: BooleanLike;
+  max_title_length: number;
+  max_description_length: number;
+};
+
+type ActiveBets = {
+  name: string;
+  description: string;
+  owner: BooleanLike;
+  creator: string;
+  current_bets: CurrentBets[];
+  locked: BooleanLike;
+};
+
+type CurrentBets = {
+  option_name: string;
+  total_amount: number;
+  personally_invested: number;
+};
+
+export const NtosSportsBetting = () => {
+  const { act, data } = useBackend<Data>();
+  const { bank_name, bank_money, can_create_bet } = data;
+  return (
+    <NtosWindow width={500} height={620}>
+      <NtosWindow.Content scrollable>
+        <Section title="User Information">
+          <Stack>
+            <Stack.Item mr={1.5}>
+              <Icon
+                name="id-card"
+                size={3}
+                mr={1}
+                color={bank_name ? 'green' : 'red'}
+              />
+            </Stack.Item>
+            <Stack fill vertical>
+              <Stack.Item>Username: {bank_name}</Stack.Item>
+              <Stack.Item>Money Available: {bank_money}cr</Stack.Item>
+            </Stack>
+          </Stack>
+        </Section>
+        <PollsSection />
+        {!!can_create_bet && <BettingCreation />}
+      </NtosWindow.Content>
+    </NtosWindow>
+  );
+};
+
+export const PollsSection = () => {
+  const { act, data } = useBackend<Data>();
+  const { active_bets = [] } = data;
+  const [Winner, setWinner] = useState('');
+  return (
+    <Section>
+      {(active_bets.length &&
```

Comment:
1. this could be an explicit length check
2.  it should be a ternary with the false value first since it's just a single box